### PR TITLE
Feature core banking

### PR DESCRIPTION
**Pull request by lshock**
Adds methods for using the Core Banking API.

### CODE REVIEW

File: src/hackathon_helper/api/money_movement.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class MoneyMovement(APIBuilder):
     """
 
     def __init__(self, api_key, api_secret):
-        """This creates the object for you to interacting with money movement API.
+        """This creates the object for you to interact with money movement API.
```
Comment (robot297) [inline src/hackathon_helper/api/money_movement.py:11]: ☹️

**Comment by robot297** [comment]
I see some white space issues according to the linter. Can you address those pl0x?

@lshock

**Comment by lshock** [comment]
I thought I went through and fixed them all already? @robot297

**Comment by robot297** [comment]
I'd check the job and see what it's complaining about.  Looks like it's about whitespace.

**Review by robot297 — CHANGES_REQUESTED** [review]
Linter is upsetti.

**Review by robot297 — APPROVED** [review]

**Pull request merged**